src/ui/MessageContent/index.tsx

```diff
@@ -269,7 +269,7 @@ export default function MessageContent({
               message={message as SendableMessageType}
               userId={userId}
               isByMe={isByMe}
-              isUnavailable={(replyType === 'THREAD' && (channel?.joinedAt ?? 0 * 1000) > (message.parentMessage?.createdAt ?? 0))}
+              isUnavailable={(channel?.messageOffsetTimestamp ?? 0) * 1000 > (message.parentMessage?.createdAt ?? 0)}
```

Comment:
It seems `* 1000` no needed
<img width="414" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/88558650-9f9d-4bce-9682-42dda4dd2c98">

Reply:
FYI
<img width="370" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/e4612ed5-1f92-406e-b4d7-949890da0e4f">

Reply:
yes you are right and we checked too! I think @HoonBaek forgot to push it

Reply:
Ah, yes.
Liam and I found it after I pushed this PR, and I haven't updated it yet.
Thanks for reminding me!